covid_moonshot/core.py

```diff
@@ -78,6 +77,8 @@ class RunDetails:
     start_smiles: str
     start_title: str
     target: str
+    end_pIC50: float = None
+    start_pIC50: float = None
```

Comment:
Nice! Sorry about this, I was worried we might have a problem here but convinced myself it was ok.

BTW, if we make the type `Optional[float]` here, we'll get warnings from mypy when we try to use `start_pIC50` without first checking if it is `None`.